examples/creatives/banner_nonlinear.js

```diff
@@ -0,0 +1,51 @@
+class BannerNonLinear extends BaseSimidCreative{
+    constructor() {
+        super();
+        this.addButtonClickActions_();
+    }
+
+    /** @override */
+    onInit(eventData) {
+        super.onInit(eventData);
+        this.dynamicResize_();
```

Comment:
you could rename this to, requestInitialSizeAndPlace.

Reply:
Other override methods in the linear creatives keep the same function name as the function from the parent class, wanted to stay consistent with that standard here as well